docker/service/Dockerfile

```diff
@@ -1,11 +1,11 @@
-FROM gradle:8.8-jdk17 AS build
+FROM gradle:8.11.0-jdk21-jammy AS build
```

Comment:
We're using 8.8 elsewhere, so I'm a little confused why the change to 8.11 here. Was that intentional?

Was the adding of `jammy` intentional either?

Reply:
yes, you are right, I need to be consistent and use gradle 8.8.